src/extensions/default/NavigationAndHistory/html/recentfiles-template.html

```diff
@@ -0,0 +1,165 @@
+<style>
```

Comment:
As all the selectors are prepended with `mrof` (or start with `#mrof-container` ID) I don't think there should be anything that touches anything outside of the MROF-list :memo: